src/cljs/athens/devcards/athena.cljs

```diff
@@ -263,12 +267,29 @@
 
 (defn results-el
   []
-  [:div (use-style results-heading-style)
-   [:h5 "Results"]
-   [:span (use-style hint-style)
-    "Press "
-    [:kbd "shift + enter"]
-    " to open in right sidebar."]])
+  (let [show-recent? @(subscribe [:show-recent?])
+        recent-item-length 40
+        style-display {:style {:display (if show-recent? "block" "none")}}]
+    [:<> [:div (use-style results-heading-style)
+          [:h5 (if show-recent? "Recent" "Results")]
+          [:span (use-style hint-style)
+           "Press "
+           [:kbd "shift + enter"]
+           " to open in right sidebar."]]
+     [:div (merge (use-style results-list-style) style-display)
```

Comment:
i think conditional rendering the recent items is a bit tricky here. The recent items should be rendered based on if the recent items list is empty or not or when query term is not `nil`. If the query term is stored in an atom that the `results-el` has to or can subscribe to from re-frame db then it can be done if not then there isn't a way to identify which state it is in.

Reply:
i took a look at code again and the query term is stored in an atom that can be passed to `results-el`.